.env

```diff
@@ -0,0 +1,9 @@
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=123456
+POSTGRES_DATABASE=blog
+POSTGRES_LOCAL_PORT=5434
+POSTGRES_DOCKER_PORT=5432
+
+SPRING_LOCAL_PORT=8088
+SPRING_DOCKER_PORT=8080
+DEBUG_PORT=5005
```

Comment:
Don’t push the `.env` file with sensitive data. better create `.env.templete`.
Let’s add `.env` to `.gitignore`